### ISSUE TITLE
Timeline plays Key Entries

### ISSUE THREAD

**Issue by mechPenSketch** [dialogic-godot/dialogic#1922]
## The problem

**Describe the bug**
When a timeline is played, it displays text under the key instead of the default language.

**To Reproduce**
When the scene plays, call `Dialogic.start(timeline)` where `timeline` is a filepath.

**Expected behavior**
The timeline should play English text by default.

**Screenshots**
![dialogic_key_text](https://github.com/coppolaemilio/dialogic/assets/15977859/8bf17f9c-321c-4c2b-94de-9cb6b6eb06fb)

**System (please complete the following information):**
 - OS: Windows 10
 - Godot Version: v4.1.2.stable.official [399c9dc39]
 - Dialogic Version: 2.0-Alpha-11 (Godot 4.1+)

## Solutions

**Workaround**
None.

**Possible fixes**
I thought it could be resolved by setting the default language to "en":
![dialogic_key_text2](https://github.com/coppolaemilio/dialogic/assets/15977859/b36304ce-2ed0-4925-ba77-4363464cb6a0)

So far, I could only think that there has to be a language settings I forgot to set somewhere.

**Comment by CakeVR** [comment]
Your generated CSV must be broken.
Can you explain the settings you used for the translation?

**Comment by mechPenSketch** [comment]
IIRC, I have checked "Enable Translations", set Default Language to `en` and File Mode to Per Timeline. As I was using the previous version of Dialogic, .cvs files are automatically generated next to timeline files. 

I thought it could be because I was using the old version of Dialogic, so I brought it up to the latest, but it still didn't work.

That said, I have finally resolved this issue by pressing "Collect Translations", but not before following its instructions on its nearby tooltip. According to the Information below, I supposed to Update by pressing "Update CSV Files", but it was disabled.
